documentation/src/main/asciidoc/introduction/Introduction.adoc

```diff
@@ -186,15 +186,15 @@ dependencies {
     annotationProcessor 'org.hibernate.orm:hibernate-processor:{fullVersion}'
 
     // Hibernate Validator
-    implementation 'org.hibernate.validator:hibernate-validator:9.0.0.Final'
+    implementation 'org.hibernate.validator:hibernate-validator:9.0.1.Final'
     implementation 'org.glassfish.expressly:expressly:6.0.0'
 
     // Agroal connection pool
     runtimeOnly 'org.hibernate.orm:hibernate-agroal:{fullVersion}'
-    runtimeOnly 'io.agroal:agroal-pool:2.5'
+    runtimeOnly 'io.agroal:agroal-pool:2.7.1'
 
     // logging via Log4j
-    runtimeOnly 'org.apache.logging.log4j:log4j-core:2.24.1'
+    runtimeOnly 'org.apache.logging.log4j:log4j-core:2.24.3'
```

Comment:
Hi @gavinking  🙂, I noticed this PR and thought I'd share...  in Search, we have these acsidoc attributes set up in the build:

https://github.com/hibernate/hibernate-search/blob/4866b35a5bea50e4f53dfa9ebd712c0452a7f49d/documentation/pom.xml#L377-L379

so that then in the asciidocs we could reference them as:

https://github.com/hibernate/hibernate-search/blob/34dde106a3f25630d37bab3f7f227194dd6ce27f/documentation/src/main/asciidoc/public/reference/_integrations.adoc?plain=1#L36

Maybe it's worth doing something similar in ORM, so there's no need to remember about updating the adoc files when someone bumps a dependency version (particularly that now there's dependabot enabled for ORM)?

Reply:
Yeah, that would be awesome 👍